lib/internal/validators.js

```diff
@@ -34,10 +34,20 @@ const {
 } = require('internal/util/types');
 const { signals } = internalBinding('constants').os;
 
+/**
+ * @callback isInt32
+ * @param {number} value
+ * @returns {boolean}
+ */
```

Comment:
```suggestion
 */
/** @type {isInt32} */
```

Reply:
This change throws an error on WebStorm

<img width="699" alt="Screen Shot 2022-08-17 at 9 51 13 AM" src="https://user-images.githubusercontent.com/1935246/185151351-338c0ab6-5b90-4fe6-9990-70d566576822.png">